tests/clickhouse-test

```diff
@@ -267,6 +268,9 @@ def run_single_test(args, ext, server_logs_level, client_options, case_file, std
         os.system("LC_ALL=C sed -i -e 's|/auto_{{shard}}||g' {file}".format(file=stdout_file))
         os.system("LC_ALL=C sed -i -e 's|auto_{{replica}}||g' {file}".format(file=stdout_file))
 
+    # Normalize hostname in stdout, stderr files.
```

Comment:
Comment about out and err files, but you change only one file. Is it as expected?

Reply:
It's not. Fixed.